### ISSUE TITLE
Changing plain text font changes context menu font

### ISSUE THREAD

**Issue by xypron** [giuspen/cherrytree#1495]
**Version, Operative system**
0.99.29, Debian bullseye

**Describe the bug**
Changing the font for plain text (Edit -> Preferences -> Fonts -> Plain Text) to Oxygen Mono Regular 12 also changed the font of the context menu when right clicking on the text pane.

**To Reproduce**
Change the font for plain text (Edit -> Preferences -> Fonts -> Plain Text) to Oxygen Mono Regular 12.
Restart CherryTree
Create a plain text node
Right click into the text pane

**Screenshots**
![Screenshot](https://user-images.githubusercontent.com/2747104/105175096-c6aad700-5b23-11eb-93e0-fe70754e8587.jpg)

**Comment by iclementine** [comment]
Same issue, from 0.99.53

**Comment by giuspen** [comment]
strange I don't see it on latest linux mint cinnamon.
@iclementine you also on Debian 11 Bullseye? Which desktop? Is it happening with every font or only Oxygen

**Comment by iclementine** [comment]
> strange I don't see it on latest linux mint cinnamon. @iclementine you also on Debian 11 Bullseye? Which desktop? Is it happening with every font or only Oxygen

No. I used cherrytree on openSUSE tumbleweed with plasma desktop 5.26.5. It happens with every font. But I fixed it using the method in this Issue. https://github.com/giuspen/cherrytree/issues/1209
